aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -347,6 +347,7 @@ module std::features {
         });
     }
 
+    #[view]
```

Comment:
Does this work if this function is private?

Reply:
Yeah see the test plan.

Reply:
Might as well make this function public. No reason for it to be private IMO :)